### PR TITLE
504: change installation steps description to make it work

### PR DESCRIPTION
**Pull request by imaffe**
### Motivation

Update docs for the [code PR #542](https://github.com/streamnative/function-mesh/pull/542)

### CODE REVIEW

File: docs/install-function-mesh.md [hunk 1]
```diff
@@ -142,14 +142,29 @@ This example shows how to install Function Mesh through [Helm](https://helm.sh/)
 - This example shows how to verify whether Function Mesh can run properly.
 
     After installing the Function Mesh Operator and deploying a Pulsar cluster, you can submit a sample CRD to create Pulsar Functions, source, sink, or Function Mesh. In this example, a CRD is submitted to the Pulsar cluster for creating a Pulsar Function. You can also submit other CRDs under the `./config/samples` directory.
-
-    1. Submit a sample CRD to the Pulsar cluster.
+    
+    1. Change the CRD definition to point to your Pulsar cluster.
```
Comment (Huanli-Meng) [inline docs/install-function-mesh.md:146]:
```suggestion
    1. Update your Pulsar cluster configurations in the CRD.
```

**Review by Huanli-Meng — APPROVED** [review]
Only one minor comment

**Comment by Huanli-Meng** [comment]
One more question: do we need to update docs for previous releases? Thanks.

**Comment by Huanli-Meng** [comment]
Ping @freeznet or @tpiperatgod for code owner review.

**Review by freeznet — APPROVED** [review]

**Pull request merged**